### PR TITLE
Node consistency 91 100

### PR DESCRIPTION
**Pull request by martinstacey**
### Purpose
The following PR addresses nodes' names and descriptions to make them more descriptive for beginners.

**91) NODE: Image.WriteToFile**
-CHANGES:
Output Name: success
Output result= was not working, added a message: File has been succesfully written

**92) NODE: Image.Dimensions**
-CHANGES:
Output Descriptions: 
Width: width of image in pixels
Height: height of image in pixels

**93) NODE: FileSystem.WriteText**
-CHANGES:
Output Name: success
Output Description: Success message if file was written correctly 
Output result=  added a message: File has been succesfully written

**94) NODE: FileSystem.ReadText**
-CHANGES:
Output : str to string
InputDescription: file: File object to read text from

**95) NODE: Directory From Path**
-CHANGES:
Output Description : Path to the directory. (string)

**96) NODE: Directory.StringifyJSON**
-CHANGES:
Output Description : Path to the directory.

**97) NODE: Directory.ParseJSON**
-CHANGES:
Output Description : Path to the directory.

**98) NODE: Cuboid.Width**
-FYI Node located on protogeometry.dll Suggestion: Add description for cuboid input and revise overall description for clarity.

**99) NODE: Cuboid.Length**
-FYI Node located on protogeometry.dll Add description for cuboid input and revise overall description for clarity.

**100) NODE: Cuboid.Height**
-FYI Node located on protogeometry.dll Add description for cuboid input and revise overall description for clarity.

image

### Reviewers
@SHKnudsen
@StudioLE
@MarkThorley
@anrova

### FYIs
-Cuboid.Width, Cuboid.Length, Cuboid.Height: located on protogeometry.dll
-Icons added: 

image

### CODE REVIEW

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs [hunk 1]
```diff
@@ -64,4 +64,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("2.8.0.2015")]
+[assembly: AssemblyFileVersion("2.9.0.2513")]
```
Comment (SHKnudsen) [inline src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs]: Shouldnt commit this file

File: src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx [hunk 2]
```diff
@@ -183,7 +183,7 @@
     <value>Creates a directory object from a path</value>
   </data>
   <data name="DirectoryObjectPortDataPathToolTip" xml:space="preserve">
-    <value>Path to the directory.</value>
+    <value>Path to the directory. (string)</value>
```
Comment (SHKnudsen) [inline src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx]: Still really dont like this... Is this something you agreed with ADSK?

File: src/Libraries/CoreNodes/FileSystem.cs [hunk 3]
```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Linq;
 using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.WindowsRuntime;
```
Comment (SHKnudsen) [inline src/Libraries/CoreNodes/FileSystem.cs]: Do we know why this is added?

Comment (martinstacey) [reply src/Libraries/CoreNodes/FileSystem.cs]: addressed in 4f79b62

File: src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx (hunk 2 above)
Comment (martinstacey) [reply src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx]: the comment on this one was to make sure that the user knows the datatype needed for the input `path`
Im changing this to the style that was suggested for other nodes. `type: string (Path to the directory)`
in 01eb3dd

**Comment by martinstacey** [comment]
Suggestions for nodes locked in protogeometry:

**Cuboid.Width**
Input description: A cuboid
Output description: cuboid's width

**Cuboid.Length**
Input description: A cuboid
Output description: cuboid's length

**Cuboid.Height**
Input description: A cuboid
Output description: cuboid's height

PS: these may seem redundant descriptions but its less scary for beginners than the default `Autodesk.DesignScript.Geometry.Cuboid` description

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs (hunk 1 above)
Comment (martinstacey) [reply src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs]: Addressed in fbdffac

**Pull request merged**